code/controllers/subsystem/pai.dm

```diff
@@ -109,6 +121,9 @@ SUBSYSTEM_DEF(pai)
 	dat += "<tr class=\"d0\"><td>Name:</td><td>[candidate.name]</td></tr>"
 	dat += "<tr class=\"d1\"><td><a href='byond://?src=[REF(src)];option=name;new=1;candidate=[REF(candidate)]'>\[Edit\]</a></td><td>What you plan to call yourself. Suggestions: Any character name you would choose for a station character OR an AI.</td></tr>"
 
+	dat += "<tr class=\"d0\"><td>Color:</td><td>[get_theme_from_color(candidate.color)] <svg width=\"20\" height=\"20\"><rect width=\"20\" height=\"20\" style=\"fill:[candidate.color];stroke-width:2;stroke:rgb(255,255,255)\" /></svg> ([candidate.color])</td></tr>"
```

Comment:
Why not store the canidate's theme in a seperate var

Reply:
Why yes?

Reply:
what do you mean?